src/runtime/pay-flow.js

```diff
@@ -143,6 +143,7 @@ export class PayStartFlow {
       true,
       getEventParams(swgPaymentRequest['skuId'])
     );
+    PayCompleteFlow.waitingForGpay_ = true;
```

Comment:
To be pedantic, this variable is being set a bit earlier than actual gpay invocation. But I suppose it's inconsequential.

Reply:
Should I change the flag to "waitingForPayClient_"?

Reply:
sure